test/parallel/test-child-process-spawn-close-stdin.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/25131
+const common = require('../common');
+const cp = require('child_process');
+const fs = require('fs');
+
+if (process.argv[2] === 'child') {
+  fs.closeSync(0);
```

Comment:
Any reason to use `closeSync(0)` rather than `process.stdin.end()`? If so, can you add a comment?